src/app/app.component.ts

```diff
@@ -14,6 +14,6 @@ export class AppComponent {
   constructor(private gameService: GameService) {}
 
   ngOnInit() {
-    this.gameService.newGame();
+    // this.gameService.newGame();
```

Comment:
habéis comentado la linea de empezar un nuevo juego, esto debería estar descomentado verdad?

Reply:
sii